webapp/src/store/getters.ts

```diff
@@ -0,0 +1,26 @@
+import { GetterTree } from 'vuex';
+import { AppState } from '../types';
+
+const getters: GetterTree<AppState, AppState> = {
+  totalAmountDonated: ({ transactions }) => {
+    return transactions.filter(t => t.type == 'donation' && t.amount > 0)
+      .map(t => t.amount)
+      .reduce((a, b) => a + b, 0);
+  },
```

Comment:
The correct filter should be `t.type == 'donation' && t.to == user._id && t.status == 'success'`

i.e. only successful transactions should count.

Reply:
Also, use `===` for equality tests.